proto/stream_plan.proto

```diff
@@ -7,9 +7,60 @@ import "common.proto";
 import "data.proto";
 import "expr.proto";
 import "plan_common.proto";
+import "source.proto";
 
 option optimize_for = SPEED;
 
+message StopMutation {
+  repeated uint32 actors = 1;
+}
+
+message DispatcherMutation {
+  uint32 actor_id = 1;
+  uint64 dispatcher_id = 2;
+  repeated common.ActorInfo info = 3;
+}
+
+message UpdateMutation {
+  repeated DispatcherMutation mutations = 1;
+}
+
+message AddMutation {
+  repeated DispatcherMutation mutations = 1;
+  map<uint32, source.ConnectorSplits> actor_splits = 2;
+}
+
+message SourceChangeSplitMutation {
+  map<uint32, source.ConnectorSplits> actor_splits = 2;
+}
+
+message AddDispatcherMutation {
```

Comment:
Would you please doc when each mutation will be used?

Reply:
Sure. I'll do this along with refactoring the `AddOutput` mutation in next PRs.